spec/views/session_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe 'Login', type: :feature do
+  describe 'login' do
+    before(:each) do
+      User.create! name: 'Sadiq Habil', password: '215234', email: 'ingqabil@gmail.com', confirmed_at: Time.now
+    end
+    it 'shows the right content' do
+      visit new_user_session_path
+      expect(page).to have_content('Log in')
+      expect(page).to have_content('Email')
+      expect(page).to have_content('Password')
+    end
+
+    it 'Fill email and password inputs' do
+      visit new_user_session_path
```

Comment:
- [ ] (Optional) The subject of this test case should reflect that the wrong user login details were filled. Having descriptions that are easy to understand makes it easier for other programmers to know what you are trying to achieve in a test. 👍